dev/com.ibm.ws.security.mp.jwt_fat/fat/src/com/ibm/ws/security/mp/jwt/fat/MPJwtConfigUsingBuilderTests.java

```diff
@@ -0,0 +1,1087 @@
+     * All other will skip the restore by overriding the restoreTestServers method
+     */
+    @Override
+    public void restoreTestServers() {
+        Log.info(thisClass, "restoreTestServersWithCheck", "* Skipping server restore **");
+        logTestCaseInServerLogs("** Skipping server restore **");
+    }
+
+    /**
+     * Gets the resource server up and running.
+     * Sets properties in bootstrap.properties that will affect server behavior
+     * Sets up and installs the test apps
+     * Adds the server to the serverTracker (used for server restore and test class shutdown)
+     * Starts the server using the provided configuration file
+     * Saves the port info for this server (allows tests with multiple servers to know what ports each server uses)
+     * Allow some failure messages that occur during startup (they're ok and doing this prevents the test framework from failing)
+     * 
+     * @param server
+     *            - the server to process
+     * @param configFile
+     *            - the config file to start the server with
+     * @param jwkEnabled
+     *            - do we want jwk enabled (sets properties in bootstrap.properties that the configs will use)
+     * @throws Exception
+     */
+    protected static void setUpAndStartRSServerForTests(LibertyServer server, String configFile, boolean jwkEnabled) throws Exception {
+        bootstrapUtils.writeBootstrapProperty(server, MpJwtFatConstants.BOOTSTRAP_PROP_FAT_SERVER_HOSTNAME, getServerHostName());
+        bootstrapUtils.writeBootstrapProperty(server, MpJwtFatConstants.BOOTSTRAP_PROP_FAT_SERVER_HOSTIP, getServerHostIp());
+        if (jwkEnabled) {
+            bootstrapUtils.writeBootstrapProperty(server, "mpJwt_keyName", "");
+            bootstrapUtils.writeBootstrapProperty(server, "mpJwt_jwksUri", getServerSecureUrlBase(jwtBuilderServer) + "jwt/ibm/api/defaultJWT/jwk\"");
+        } else {
+            bootstrapUtils.writeBootstrapProperty(server, "mpJwt_keyName", "rsacert");
+            bootstrapUtils.writeBootstrapProperty(server, "mpJwt_jwksUri", "");
+        }
+        deployRSServerApiTestApps(server);
+        serverTracker.addServer(server);
+        server.startServerUsingExpandedConfiguration(configFile);
+        saveServerPorts(server, MpJwtFatConstants.BVT_SERVER_1_PORT_NAME_ROOT);
+        server.addIgnoredErrors(Arrays.asList(MpJwtMessageConstants.CWWKW1001W_CDI_RESOURCE_SCOPE_MISMATCH));
+    }
+
+    //    protected static void setUpAndStartBuilderServer(LibertyServer server, String configFile, boolean jwtEnabled, String builderApp) throws Exception {
+    //        deployBuilderServerComplexBuilderApp(server);
+    //        setUpAndStartBuilderServer(server, configFile, jwtEnabled);
+    //    }
+
+    /**
+     * Initialize the list of test application urls and their associated classNames
+     * 
+     * @throws Exception
+     */
+    protected static void initTestAppArray() throws Exception {
```

Comment:
This shouldn't have to be a `static` method - it could be a normal class method slightly modified to return a `List<List<String>>` that then gets called by `genericConfigTest()`.